fr-portfolio.html

```diff
@@ -0,0 +1,80 @@
+<!DOCTYPE html>
+<html lang="en-gb">
+<head>
+	<link rel="stylesheet" type="text/css" href="normalize.css">
+	<link rel="stylesheet" type="text/css" href="portfolio.css">
+	<link rel="stylesheet" href="http://maxcdn.bootstrapcdn.com/font-awesome/4.3.0/css/font-awesome.min.css">
+	<title>Freddie's Portfolio</title>
+</head>
+<body>	
+ <main>	
+  <nav class="navbar">
+   <a href="#portfolio">Portfolio</a>
+   <a href="#about">About</a>
+   <a href="#contact">Contact</a>
+  </nav>
+	
+  <section class="welcome-banner" >
+   <h1>Freddie's Portfolio</h1>
+  </section>	
+	
+  <article id="about" class="about-alt">
+	<h2> About me </h2>
+		<div class="about-me-container">
+			<figure>
+				<img src="https://via.placeholder.com/350x200">
+			</figure>
+			<div>
+				<p>Hi this is where I should put some information about myself</p>
+			</div>
+		</div>		
+  </article>
+		
+  <section id="portfolio" class="projects">
+   <h2>Portfolio</h2>
+   <div class="portfolio-container"> <!-- Portfolio container left hand start -->
+     <figure>
+		<img src="https://via.placeholder.com/350x200">
+		<figcaption>This is where my project information will go</figcaption>
```

Comment:
`alt="something descriptive"` needs to be added for the `<img>`, `<figcaption>` is just to add a small bit of text to an image rather than contain long bits of text / information.

Reply:
alt has been added. figcaption has been changed to p tags.